jupyterlab_gcpscheduler/src/components/dialog.tsx

```diff
@@ -170,54 +158,21 @@ export class SchedulerDialog extends React.Component<Props, State> {
   }
 
   render() {
-    const projectId = this.getProjectId();
     return (
       <Dialog open={this._isOpen()} scroll="body">
         {this.state.showCreateForm && (
           <header className={localStyles.header}>
             <span className={localStyles.title}>
-              Create a notebook{' '}
-              {this.state.creatingExecution ? 'execution' : 'schedule'}{' '}
+              Submit notebooks to Executor
```

Comment:
*nit* Should this be notebook since we're only submitting one file for execution at a time?

Reply:
I get your point. I'll change it and let Huaiwei know.